### PR TITLE
Update AUTHORS

### PR DESCRIPTION
**Pull request by EPuncker**
### THIS PR WILL BE OPEN FOR A WEEK (WILL BE MERGED ON 15/03)

**Everyone feel free to request changes or whatsoever**

### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- Updated to include everyone's name
- Added OpenTibia developers
- Removed emails

Sorry to anyone that was left out last time, it was not intentional and was made in a hurry, and for future reference, this is the one that is going to be used from now onwards:
`git log --all --format='%aN' | sort -u | uniq > AUTHORS`

**Issues addressed:** 

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by EPuncker** [comment]
THIS PR WILL BE OPEN FOR A WEEK (WILL BE MERGED ON 15/03)

Everyone fell free to request changes and whatsoever

**Review by Erza — APPROVED** [review]

File: AUTHORS [hunk 1]
```diff
@@ -1,70 +1,197 @@
-The Forggoten Server contributors/developers
+The Forggoten Server Contributors/Developers
```
Comment (ramon-bernardo) [inline AUTHORS:1]: Forgotten?

Comment (ramon-bernardo) [inline AUTHORS:1]: Forgotten?

**Comment by ranisalt** [comment]
Idea: maybe we could add that to the wiki instead of versioning it, then we can add a link in README

**Review by marksamman — DISMISSED** [review]

File: AUTHORS [hunk 2]
```diff
@@ -1,70 +1,197 @@
...
 nclx <januario.nclx@gmail.com>
+Nekiro <nekiro.dev@gmail.com>
 octavio007 <octavioargenis_11@hotmail.com>
+Patryk Zajdler <pzajdler2@gmail.com>
 paulosalvatore <paulo.salvatore@hotmail.com.br>
+Pegason <49088829+pegason@users.noreply.github.com>
 PrinterLUA <roberto_potrus@hotmail.com>
-Ranieri Althoff <ranisalt@gmail.com>
+Ramon Bernardo <ramonbernardo.contato@gmail.com>
+Ranieri Althoff <ranisalt@protonmail.com>
```
Comment (ranisalt) [inline AUTHORS:107]:
```suggestion
Ranieri Althoff <1993083+ranisalt@users.noreply.github.com>
```

How did you get that email? :thinking:

File: AUTHORS [hunk 3]
```diff
@@ -1,70 +1,197 @@
-The Forggoten Server contributors/developers
+The Forgotten Server Contributors/Developers
 
 Allan Ference <f.fallen45@gmail.com>
+andremiles1 <andre_lod@hotmail.com>
```
Comment (ranisalt) [inline AUTHORS:4]:
```suggestion
Andre Miles <andre_lod@hotmail.com>
```

cc @andremiles1

**Review by ranisalt — CHANGES_REQUESTED** [review]

File: AUTHORS (hunk 2 above)
Comment (EPuncker) [reply AUTHORS:107]: > How did you get that email? 🤔

only GitHub knows 😄 
`git log --all --format='%aN <%aE>' | sort -u | uniq > AUTHORS`

**Review by DSpeichert — APPROVED** [review]

**Comment by ranisalt** [comment]
> `git log --all --format='%aN <%aE>' | sort -u | uniq > AUTHORS`

This has changed, right? You're not exporting the emails anymore.

**Comment by EPuncker** [comment]
> > `git log --all --format='%aN <%aE>' | sort -u | uniq > AUTHORS`
> 
> This has changed, right? You're not exporting the emails anymore.

yeah fixed the PR description, should be merged now

**Comment by EPuncker** [comment]
@ranisalt accept&merge please

**Comment by ranisalt** [comment]
> @ranisalt accept&merge please

sorry I was sleeping

**Pull request merged**